core/common/src/main/java/alluxio/security/authorization/PermissionStatus.java

```diff
@@ -86,52 +90,69 @@ public FileSystemPermission getPermission() {
    *
    * @param umask the umask to apply
    * @return a new {@link PermissionStatus}
-   * @see FileSystemPermission#applyUMask(FileSystemPermission)
    */
   public PermissionStatus applyUMask(FileSystemPermission umask) {
     FileSystemPermission newFileSystemPermission = mPermission.applyUMask(umask);
     return new PermissionStatus(mUserName, mGroupName, newFileSystemPermission);
   }
 
   /**
-   * Gets the Directory default {@link PermissionStatus}. Currently the default dir permission is
-   * 0777.
+   * Applies default umask to new created files.
    *
-   * @return the default {@link PermissionStatus} for directories
+   * @param conf the runtime configuration of Alluxio
+   * @return a new {@link PermissionStatus}
+   */
+  public PermissionStatus applyFileUMask(Configuration conf) {
```

Comment:
change pushed.  the method is no more a static one, but modifying the member variables.